### DISCUSSION TITLE
npm install antd --save出现warn，然后项目启动报错

### DISCUSSION THREAD

**Discussion by githubqh** [ant-design/ant-design-mobile#5345] [Q&A]
![image](https://user-images.githubusercontent.com/105548468/174216035-fec68646-f93a-4976-93a3-5194678005a9.png)
![image](https://user-images.githubusercontent.com/105548468/174216046-b4a8abb0-ede5-4944-8a4a-f6fa486cfd35.png)

**Comment by miracles1919** [comment]
包弄错了... `npm i antd-mobile`

**Reply by githubqh** [reply]
> 包弄错了... `npm i antd-mobile`
没有，我是需要mobile和web端的antd两个包的

**Comment by awmleer** [comment]
最好把你的项目上传到 GitHub 上或者打个 zip 发到这里，不然我们没有办法帮你排查
